workspace-manager-clienttests/src/main/resources/serviceaccounts/dev-workspace-app-sa.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "workspace-dev@broad-dsde-dev.iam.gserviceaccount.com",
```

Comment:
please absolutely get dedicated SAs for these, instead of reusing existing ones and allowing their credentials to land on many people's laptops.  I'd love to see a dedicated SA for delegate-user-sa.json too

Reply:
This is something we will need to work in DevOps, still in progress. At some point we will create a new PR for this.